discretize/View.py

```diff
@@ -618,6 +618,8 @@ def plotGrid(
                     marker="^", linestyle=""
                 )
 
+            line_color = kwargs.get('line_color', 'C0')
+            line_width = kwargs.get('line_width', 1.)
```

Comment:
I am not sure if these are the best keywords. In some ways, I think we might want to re-write this function as a class that has properties which are the line styles, etc. Thoughts?

Reply:
keeping the names the same as matplotlib would be useful 
so linewidth, not line_width
line_color would just be color

I also notice in Utils.PlotUtils.plot2Ddata that the default line width is 2, which is too fat (i've changed my version to 1).